scripts/fdbshow

```diff
@@ -111,15 +112,20 @@ class FdbShow(object):
                 if 'bvid' not in fdb:
                     # no possibility to find the Vlan id. skip the FDB entry
                     continue
-                try:
-                    vlan_id = port_util.get_vlan_id_from_bvid(self.db, fdb["bvid"])
-                    if vlan_id is None:
-                        # the situation could be faced if the system has an FDB entries,
-                        # which are linked to default Vlan(caused by untagged trafic)
-                        continue
-                except Exception:
-                    vlan_id = fdb["bvid"]
-                    print("Failed to get Vlan id for bvid {}\n".format(fdb["bvid"]))
+                bvid = fdb["bvid"]
+                if bvid in bvid_tlb:
+                    vlan_id = bvid_tlb[bvid]
+                else:
+                    try:
+                        vlan_id = port_util.get_vlan_id_from_bvid(self.db, bvid)
+                        if vlan_id is None:
```

Comment:
>if vlan_id is None: [](start = 24, length = 19)

Even `if vlan_id is None`, you can still cache it to save future redis access.

Reply:
Agree, updated patch